charts/cmc-claim-store/values.preview.template.yaml

```diff
@@ -35,8 +35,6 @@ java:
     IDAM_API_URL: https://idam-api.aat.platform.hmcts.net
     PDF_SERVICE_URL: http://cmc-pdf-service-aat.service.core-compute-aat.internal
     DOCUMENT_MANAGEMENT_URL: http://dm-store-aat.service.core-compute-aat.internal
-    PAY_URL: false
-    FEES_URL: false
```

Comment:
Please don't remove these

Reply:
same as in aat. It's default in values.yaml file already ...

Reply:
These are the default values in `values.yaml`, we should have only those here which we want to override defaults.